tests/mock_classes/llm_chain.py

```diff
@@ -0,0 +1,18 @@
+def mock_llm_chain(retval):
+    class MockLLMChain:
+        """
+        Mock LLMChain class for testing
+
+        Example usage in a test:
+
+        monkeypatch.setattr(langchain.chains, "LLMChain",
+            MockLLMChain(desired_response='9'))
```

Comment:
desired_response isn't an arg name, the argname is retval, so should this be 

```suggestion
            MockLLMChain(retval='9'))
```

instead?